static/app/views/issueDetails/allEventsTable.tsx

```diff
@@ -24,15 +25,37 @@ export interface Props {
   excludedTags?: string[];
 }
 
+const makeGroupPreviewRequestUrl = ({groupId}: {groupId: string}) => {
+  return `/issues/${groupId}/events/latest/`;
+};
+
 function AllEventsTable(props: Props) {
   const {location, organization, issueId, excludedTags, group} = props;
   const config = getConfigForIssueType(props.group);
   const [error, setError] = useState<string>('');
   const routes = useRoutes();
   const {fields, columnTitles} = getColumns(group, organization);
 
+  const endpointUrl = makeGroupPreviewRequestUrl({
+    groupId: group.id,
+  });
+
+  const queryEnabled = group.issueCategory === IssueCategory.PERFORMANCE;
+  const {data, isLoading, isLoadingError} = useApiQuery<EventTransaction>([endpointUrl], {
+    staleTime: 60000,
+    enabled: queryEnabled,
```

Comment:
the problem here was that `isLoading` was set to `true` even though the query was never ran

Reply:
it looks like it's a react query default behavior